lib/generators/deimos/active_record/templates/migration.rb.tt

```diff
@@ -0,0 +1,24 @@
+class <%= migration_class_name %> < ActiveRecord::Migration<%= migration_version %>
+  def up
+    return if table_exists?(:<%= table_name %>)
+    create_table :<%= table_name %> do |t|
+      <%- fields.each do |key| -%>
+      <%- next if %w(id message_id timestamp).include?(key.name) -%>
+      <%- sql_type = schema_base.sql_type(key)
+         if %w(record array map).include?(sql_type)
+           sql_type = ActiveRecord::Base.connection.supports_json? ? :json : :string
+         end
+      -%>
+      t.<%= sql_type %> :<%= key.name %>
+      <%- end -%>
+    end
+
+    # TODO add indexes as necessary
+  end
+
+  def down
+    return unless table_exists?(:<%= table_name %>)
+    drop_table :<%= table_name %>
```

Comment:
We should probably add at least a warning during the `up` if the table already existed so that it doesn't silently drop if the user accidentally gave the name of an existing table